src/biome/text/dataset_readers/sequence_pair_classifier_dataset_reader.py

```diff
@@ -48,3 +48,4 @@ def text_to_instance(
 
 # Register an alias for this reader
 DatasetReader.register("similarity_classifier")(SequencePairClassifierReader)
+DatasetReader.register("biome_bimpm")(SequencePairClassifierReader)
```

Comment:
It this line an error? Do we need define this line? It's not managed by corresponding biome pipeline class?